.github/workflows/release.yml

```diff
@@ -15,11 +15,11 @@ jobs:
         if: ${{ !github.event.client_payload.version }}
         run: |
           DATE_PART=$(date +'%y.%-m')
-          PATCH_VERSION=0
+          declare -i PATCH_VERSION=0
           while curl -sf -o /dev/null "https://api.github.com/repos/$GITHUB_REPOSITORY/git/ref/tags/$DATE_PART.$PATCH_VERSION"; do
-            (( PATCH_VERSION++ ))
+            PATCH_VERSION+=1
```

Comment:
This will append the string `1`, not arithmetic add 1.

Reply:
Never mind, there's a `declare -i` there.

Reply:
Good stuff.

Reply:
It won't as `PATCH_VERSION` is declared as an integer above (tested this)

Reply:
I'm not sure if github actions runs bash, though I'd expect it to. (`declare` is bash specific)

Reply:
@BYK https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#defaultsrun

Reply:
I would add this just to be extra bulletproof.

```
defaults:
  run:
    shell: bash
```

Reply:
I wouldn't expect them to change this without a huge warning as it would break so many things so I'd prefer to keep this simple.